app/src/main/res/layout/shade_screen.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Empty view groups like this should be framelayouts. They are basically linear layouts with only a single slot but they were designed for this.